src/main/webapp/app/exam/manage/suspicious-behavior/suspicious-behavior.component.html

```diff
@@ -0,0 +1,34 @@
+<h5>{{ 'artemisApp.examManagement.suspiciousBehavior.title' | artemisTranslate }}</h5>
+
+<p>
+    {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessionDescription' | artemisTranslate }}
+</p>
+
+<ul>
+    <li>
+        {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessionCriterion1' | artemisTranslate }}
+    </li>
+    <li>
+        {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessionCriterion2' | artemisTranslate }}
+    </li>
+    <li>
+        {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessionCriterion3' | artemisTranslate }}
+    </li>
+</ul>
+
+<p>
+    {{ 'artemisApp.examManagement.suspiciousBehavior.suspiciousSessions.number' | artemisTranslate : { numberOfSuspiciousSessions: suspiciousSessions.length } }}
+</p>
+
+<button [disabled]="suspiciousSessions.length === 0" type="submit" (click)="goToSuspiciousSessions()" class="btn btn-primary mb-5">
```

Comment:
Suggestion (not sure): Never disable this button and instead have an expressive empty state on the page itself. This would avoid confusion about why the button is disabled.